apis/config/scheme/scheme_test.go

```diff
@@ -1022,6 +1024,7 @@ profiles:
       apiVersion: kubescheduler.config.k8s.io/v1
       kind: CoschedulingArgs
       permitWaitingTimeSeconds: 10
+      podGroupBackoffTime: 0
```

Comment:
I'd expect without explicitly specifying it, UT would also pass. Could you check if you want to add explicit defaulting logic? (convert nil from versioned args to 0 in internal args)

Reply:
`podGroupBackoffTime: 0` means we will not backoff pod groups.
I think maybe we don't have to add explicit defaulting logic, because by default nil will be convert to 0

Reply:
That's correct. The previous comment was more about "how it behaves without explicitly setting the argument", and hence I'd like to see this UT passes w/o specifying it to 0.

Reply:
Ok, I added a test for the case

Reply:
sry for confusing, I misread - this is the test for encode.

could you apply this diff - we want to test both defaulting logic (in decode), and encode for non-default logic: https://gist.github.com/Huang-Wei/d86f188b4e85b48ad4e3e4aa4bd427da